src/notifications/ContentRules.js

```diff
@@ -16,9 +17,9 @@ limitations under the License.
 
 'use strict';
 
-const PushRuleVectorState = require('./PushRuleVectorState');
+import {PushRuleVectorState} from "./PushRuleVectorState";
 
-module.exports = {
+export class ContentRules {
```

Comment:
Same comment as https://github.com/matrix-org/matrix-js-sdk/pull/1134 on classes of static methods.